.env.example

```diff
@@ -1,17 +1,17 @@
 # Base Path Openshift: /business Firebase: /
-VUE_APP_PATH="/business"
+VUE_APP_PATH="/"
 
 #vaults Shared
 VUE_APP_ADDRESS_COMPLETE_KEY=
 
 #vaults web-url
-VUE_APP_REGISTRY_HOME_URL="https://dev.bcros.ca/"
-VUE_APP_AUTH_WEB_URL="https://dev.bcregistry.ca/business/auth/"
-VUE_APP_BUSINESSES_URL="https://dev.bcregistry.ca/business/auth/"
-VUE_APP_DASHBOARD_URL="https://dev.bcregistry.ca/business/"
+VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
+VUE_APP_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
+VUE_APP_BUSINESSES_URL="https://dev.account.bcregistry.gov.bc.ca/"
+VUE_APP_DASHBOARD_URL="https://dev.business.bcregistry.gov.bc.ca/"
 VUE_APP_SITEMINDER_LOGOUT_URL="https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
-VUE_APP_BUSINESS_CREATE_URL="https://business-create-dev.apps.silver.devops.gov.bc.ca/businesses/create/"
-VUE_APP_BUSINESS_EDIT_URL="https://business-create-dev.apps.silver.devops.gov.bc.ca/businesses/edit/"
+VUE_APP_BUSINESS_CREATE_URL="https://dev.create.business.bcregistry.gov.bc.ca/"
+VUE_APP_BUSINESS_EDIT_URL="https://dev.edit.business.bcregistry.gov.bc.ca/"
```

Comment:
I think these changes will need to be made at the same time as edit-ui and create-ui since the other repos navigate back to filings-ui.  Is that the plan?

Reply:
yes, working on it 1 by 1. You will see other two PR coming up soon.